firebase-storage/src/main/java/com/google/firebase/storage/StorageMetadata.java

```diff
@@ -398,6 +403,11 @@ public Builder setContentEncoding(@Nullable String contentEncoding) {
       return this;
     }
 
+    @Nullable
```

Comment:
Please add JavaDoc (here and everywhere). This also needs to go through API review, but an email might be fine here.

Reply:
Thanks for sending the review doc. Please do add JavaDoc though.

Reply:
done.

Reply:
@rachelsaunders Can you take a quick glance at this? Do we want to use `@return`?

Reply:
I'm actually not familiar with the syntax required here.  It's something on my learning TODOs for post-FireConf, though!

Is there a similar situation in another file that might be an exemplar for what you're trying to achieve? If not, we can reach out to some other folks to see if they know the answer.

Reply:
I think we can go down three routes here:

```
/** Returns the content encoding for the {@link StorageReference} */
```

```
/** 
   * @return The content encoding for the {@link StorageReference}.
   */
```

```
/** 
   * Returns the content encoding for the {@link StorageReference} 
   *
   * @return The content encoding for the {@link StorageReference}.
   */
```

If Rachel doesn't have a preference, we can merge as is and correct if needed when the docs are generated.

Reply:
@rachelmyers I think sebastian refers to using `@return` tag as he's doing in the rest of the file instead of writing `/** Returns ` This file uses both, so I can go with any style.

Reply:
Yes, I would agree with consistency, unless one syntax results in a different output style than the other -- _and_ those different styles are desired.

What does the rest of the Firestore library use?

FWIW - This is the ktx library, so we currently don't generate Javadocs for it... yet.

Reply:
We are very inconsistent :)

Reply:
I've updated the javadocs to use @return, since it's the most popular option in this particular file. PTAL Thanks!